app/assets/stylesheets/application.scss

```diff
@@ -48,3 +48,7 @@
 .solicitor_results_list {
   padding-left: 14px;
 }
+td.actions{
```

Comment:
Space before `{` - if I'm nitpicking Anson's and Jack's PRs then I have to do everyone's :smile: